bigquery/google/cloud/bigquery/table.py

```diff
@@ -508,14 +511,14 @@ def partitioning_type(self, value):
     def partition_expiration(self):
         """Union[int, None]: Expiration time in milliseconds for a partition.
 
-        If this property is set and :attr:`partitioning_type` is not set,
-        :attr:`partitioning_type` will default to
-        :attr:`TimePartitioningType.DAY`.
+        If :attr:`partition_expiration` is set and :attr:`partitioning_type` is
+        not set, :attr:`partitioning_type` will default to
+        :attr:`~google.cloud.bigquery.table.TimePartitioningType.DAY`.
         """
         warnings.warn(
             "This method will be deprecated in future versions. Please use "
             "Table.time_partitioning.expiration_ms instead.",
-            PendingDeprecationWarning)
+            UserWarning)
```

Comment:
What's the reason to use `UserWarning` instead of `PendingDeprecationWarning` or `DeprecationWarning`?

Reply:
This was @jonparrott's preference